filter/challenges-08.test.js

```diff
@@ -0,0 +1,288 @@
+'use strict';
```

Comment:
Make sure you're checking out `master` before you check out each new branch, so you don't get earlier files in your new branches!